OpenRA.Mods.Common/AI/SupportPowerDecision.cs

```diff
@@ -125,7 +125,7 @@ public int GetAttractiveness(Actor a, Stance stance, Player firedBy)
 				if (a == null)
 					return 0;
 
-				if (!a.IsTargetableBy(firedBy.PlayerActor))
+				if (!a.IsTargetableBy(firedBy.PlayerActor) || !a.CanBeViewedByPlayer(firedBy))
```

Comment:
This looks like it belongs in the third commit?

Reply:
Fixed.